### PR TITLE
Sidebar hyperlink, sidebar hover and meta tags

### PR DESCRIPTION
**Pull request by Sharmaryan**
added hyperlink in sidebar, hover effect in sidebar and added meta tags

### CODE REVIEW

File: styles/sidebar/sidebar.css [hunk 1]
```diff
@@ -1,17 +1,25 @@
...
   min-width: 15rem;
-  height: 80vh;
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  justify-content: space-evenly;
-   background: var(--default);
-  border: 3px solid var(--secondary-color);
   border-radius: 3%;
+  background: var(--default);
```
Comment (zabihshaik) [inline styles/sidebar/sidebar.css:4]: Better to use variable name to be clear for background, eg. default-bg

**Review by zabihshaik — APPROVED** [review]
Everything looks good. And the sidebar links are accessible now.

File: styles/sidebar/sidebar.css (hunk 1 above)
Comment (Sharmaryan) [reply styles/sidebar/sidebar.css:4]: sure, will do that

**Review by raghav6202 — COMMENTED** [review]
Didn't understand why you have used id's when you have not used them in CSS .Apart from that no queries looks good to merge. Nice U.I  👍

**Review by Himanshu-Dhawale — APPROVED** [review]

**Pull request merged**